applications/common/backend/src/test/resources/tablecell.json

```diff
@@ -1,13 +1,19 @@
 {
-  "default": {
-    "jcr:primaryType": "nt:unstructured",
-    "sling:resourceType": "kyanite/common/components/tablecell"
-  },
-  "complex": {
-    "jcr:primaryType": "nt:unstructured",
-    "sling:resourceType": "kyanite/common/components/tablecell",
-    "text": "Table cell text",
-    "rowspan": 2,
-    "colspan": 3
+  "tablebody": {
+    "tablerow1": {
+      "default": {
+        "jcr:primaryType": "nt:unstructured",
+        "sling:resourceType": "kyanite/common/components/table/tablecell"
```

Comment:
To avoid future confusion, I've fixed this resource type to be the same as in main code.
Even if in this place this is not that necessary - it's more consistent now